modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMIndexerImpl.java

```diff
@@ -667,7 +678,13 @@ protected QueryFilter createFieldValueQueryFilter(
 
 		BooleanQuery booleanQuery = new BooleanQueryImpl();
 
+		Boolean localizableField = true;
```

Comment:
Hey @davidtellolr, I think we can do the following, for example:
```
Boolean localizableField;

if (ddmStructure.hasFieldByFieldReference(fieldReference)) {
	localizableField = GetterUtil.getBoolean(
		ddmStructure.getFieldPropertyByFieldReference(
			fieldReference, "localizable"));
```
What do you think?

Reply:
Hi @natocesarrego ,

It looks fine, but we must initialize the boolean because if not, there is a compilation error in line 704.
I add an additional commit with changes. What do you think about the code now? If you prefer a single commit, I could send a new PR. 

Thanks